src/client/public/scripts/dash/dash.ts

```diff
@@ -23,56 +25,56 @@ type StatusDisplay = {
 
 const DEFAULT_AVATAR = '/images/default-avatar.svg';
 
-const dashTmpl = (data: api.DashResponse, newlyActionablePRs: string[]) => {
-  const imageUrl = data.user.avatarUrl ? data.user.avatarUrl : DEFAULT_AVATAR;
+const profileTmpl = (data: api.DashboardUser) => {
+  const imageUrl = data.avatarUrl ? data.avatarUrl : DEFAULT_AVATAR;
   const buttonTemplates = [];
 
-  if (data.user.isCurrentUser) {
+  if (data.isCurrentUser) {
     buttonTemplates.push(
         html`<a href="/settings" title="Settings" class="settings"></a>`);
   }
-  const prListTemplate =
-      (prList: api.PullRequest[],
-       newlyActionablePRs: string[],
-       emptyMessage: string) => {
-        if (prList.length) {
-          return prList.map((pr) => prTemplate(pr, newlyActionablePRs));
-        } else {
-          return html
-          `<div class="pr-list__empty-message">${emptyMessage}</div>`;
-        }
-      };
-
   return html`
-  <div class="profile-container">
     <div class="profile-avatar"><img src="${imageUrl}" alt="Avatar of ${
-      data.user.login}" /></div>
-    <div class="profile-header">Welcome ${data.user.login}</div>
+      data.login}" /></div>
+    <div class="profile-header">Welcome ${data.login}</div>
     <div class="profile-buttons">
       ${buttonTemplates}
-    </div>
-  </div>
-  <div class="pr-list">
-    <h2>Outgoing pull requests</h2>
-    ${
-      prListTemplate(
-          data.outgoingPrs,
-          newlyActionablePRs,
-          'You have no outgoing pull requests. When you open new pull requests, they\'ll appear here')}
-    <h2>Incoming pull requests</h2>
-    ${
+    </div>`;
+};
+
+const prListTemplate =
+    (prList: api.PullRequest[],
+     newlyActionablePRs: string[],
+     emptyMessage: string) => {
+      if (prList.length) {
+        return html`${prList.map((pr) => prTemplate(pr, newlyActionablePRs))}`;
+      } else {
+        return html
+        `<div class="pr-list__empty-message">${emptyMessage}</div>`;
+      }
+    };
+
+function renderOutgoing(
+    data: api.OutgoingDashResponse, newlyActionablePRs: string[]) {
+  render(
+      profileTmpl(data.user),
+      (document.querySelector('#profile-container') as Element));
+  render(
       prListTemplate(
-          data.incomingPrs,
+          data.prs,
           newlyActionablePRs,
-          '🎉 No incoming pull requests! When you\'re added as a reviewer to a pull request, it\'ll appear here.')}
-  </div>
-  `;
-};
+          'You have no outgoing pull requests. When you open new pull requests, they\'ll appear here'),
```

Comment:
Optional: Moving this to a constant would make parsing the code a little easier. (Same for PR below).

Reply:
Done.